packages/create-cloudflare/templates/next/env.d.ts

```diff
@@ -1,13 +1,7 @@
-declare global {
-  namespace NodeJS {
-    interface ProcessEnv {
-      // Add here the Cloudflare Bindings you want to have available in your application
-      // (for more details on Bindings see: https://developers.cloudflare.com/pages/functions/bindings/)
-      //
-      // KV Example:
-      // MY_KV: KVNamespace
-    }
-  }
+interface CloudflareEnv extends Env {
```

Comment:
isn't this file going to conflict with the output from `wrangler types`?

Reply:
yes good point.... let me remove it for now....

then later we can re-enable it when `wrangler types` allows us to directly set the interface name `CloudflareEnv` and the file path `env.d.ts`